test/release-notify-action.spec.js

```diff
@@ -29,6 +30,7 @@ beforeEach(() => {
   issue.createOrUpdateIssue.mockReset()
   issue.getLastOpenPendingIssue.mockReset()
   issue.closeIssue.mockReset()
+  issue.getLastClosedNotifyIssue.mockReset()
```

Comment:
we'll need considerably more testing as this feature adds complexity, so we need to test all the possible scenarios what can happen with the issue, the latest release and unreleased commits